### DISCUSSION TITLE
NGINX conf

### DISCUSSION THREAD

**Discussion by ARR4NN** [requarks/wiki#4523] [Help / Questions]
Hello, I have installed wikijs in `var/www/wiki` and I need to make a Nginx conf to get me there with the domain `docs.furnacebot.com` can you please help me where to point it too.
Thanks

**Comment by Phil-hostwiki** [comment]
Wiki.js will run on port 3000 by default. What you have to setup is Nginx reverse proxy which points to port 3000, and not `var/www/wiki`.

Here is a basic Nginx reverse proxy block;
 ```
server {
	# replace example.com with your domain name
	server_name example.com;
	
	listen 80;
	listen [::]:80;

	location / {
		proxy_pass http://127.0.0.1:3000;
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	}
}
```
